templates/settings-admin.php

```diff
@@ -54,36 +54,26 @@ function FormatMegabytes(int $byte): string {
 					<img class="infoicon" src="<?php p(image_path('core', 'actions/screen.svg')); ?>">
 					<?php p($_['hostname']); ?>
 				</h2>
-				<div class="table-wrapper">
-					<table class="server-infos-table">
-						<thead>
-						</thead>
-						<tbody>
-						<tr>
-							<td><?php p($l->t('Operating System')); ?>:</td>
-							<td><?php p($_['osname']); ?></td>
-						</tr>
-						<tr>
-							<td><?php p($l->t('CPU')); ?>:</td>
-							<td><?php p($_['cpu']) ?></td>
-						</tr>
-						<tr>
-							<td><?php p($l->t('Memory')); ?>:</td>
-							<td><?php p(FormatMegabytes($memory->getMemTotal())) ?></td>
-						</tr>
-						<tr>
-							<td><?php p($l->t('Server time')); ?>:</td>
-							<td><span class="info" id="servertime"></span></td>
-						</tr>
-						<tr>
-							<td><?php p($l->t('Uptime')); ?>:</td>
-							<td><span class="info" id="uptime"></span></td>
-						</tr>
-						</tbody>
-					</table>
-				</div>
 			</div>
-
+			<div class="col col-12">			
+				<p><?php p($l->t('Operating System').':'); ?> <strong id="numFilesStorage"><?php p($_['osname']); ?></strong></p>
+				<p><?php p($l->t('CPU').':'); ?>
```

Comment:
```suggestion
				<p><?php p($l->t('Operating System:')); ?> <strong id="numFilesStorage"><?php p($_['osname']); ?></strong></p>
				<p><?php p($l->t('CPU:')); ?>
```

etc so french translators can add their space before the colon